src/Components/CreateImageWizard/formComponents/Oscap.js

```diff
@@ -91,13 +109,61 @@ const ProfileSelector = ({ input, showSelector }) => {
         isDisabled={!isSuccess}
       >
         {isSuccess &&
-          data.map((key, index) => <SelectOption key={index} value={key} />)}
+          [<SelectOption key={0} value={'None'} />].concat(
+            data.map((key, index) => (
+              <SelectOption key={index + 1} value={key} />
+            ))
+          )}
         {isFetching && (
           <SelectOption isNoResultsOption={true} data-testid="policies-loading">
             <Spinner size="md" />
           </SelectOption>
         )}
       </Select>
+      <TextContent>
```

Comment:
I think we should only show this `<TextContent>` if a profile has been selected.